src/packages/core/icon-registry/svgs/icon-twitter.svg

```diff
@@ -0,0 +1,5 @@
+<svg fill="currentColor" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```

Comment:
Is this SVG icon being used? (I'd assume that the `icon-twitter.js` was being used instead?)

Reply:
You are right, one of them is from V13 and the other from Simpleicons. Since Simpleicons is going to remove their Twitter icon soon, I have taken the liberty to download their SVG and scrapped our V13 version.